packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -224,14 +273,25 @@ const LayoutHeader = forwardRef((props, ref) => {
             onClick={props.toggleTopMenu}
             onKeyPress={handleTopMenuButtonKeyPress}
           >
-            <SpacingsInline alignItems="center">
-              <span>{props.siteTitle}</span>
-              {props.isTopMenuOpen ? (
-                <AngleUpIcon size="medium" color="info" />
-              ) : (
-                <AngleDownIcon size="medium" />
+            <TitleContainer hasContext={!!siteContextTitle}>
+              {siteContextTitle && (
+                <SiteContextTitle>
+                  {siteContextTitle}&nbsp;&nbsp;&gt;{' '}
```

Comment:
Nit: have you tried other approaches than `&nbsp;&nbsp;&gt;{' '}`? For example using `::after`?

<img width="632" alt="image" src="https://user-images.githubusercontent.com/1110551/197134842-b500886a-f874-44ae-86b0-7d7d85c306d7.png">

Reply:
I haven't but I can give it a go. Thank you 👍